routes/logs.js

```diff
@@ -6,5 +6,6 @@ const { authorizeUser } = require("../middlewares/authorization");
 const { ROLES } = require("../constants/users");
 
 router.get("/:type", authenticate, authorizeUser(ROLES.SUPER_USER), logs.fetchLogs);
+router.get("/member/cache/self", authenticate, logs.fetchMemberCacheLogs);
```

Comment:
Can we call it `/cache/self` or something that doesn't need so much nesting?

Reply:
`/CACHE_SELF` I'd recommend

Reply:
We can change it to `/cache/self` but I was maintaining the consistency for which this API is specifically made
Reference here : https://github.com/Real-Dev-Squad/website-backend/blob/b2cb0dfd3bbde7af33f320cd9c0681af45ae2301/routes/logs.js

Reply:
It's regarding that only. `/:type`, is the type of log in full capital letters. Eg - /PROFILE_DIFF_APPROVED, /PROFILE_HEALTH

Reply:
Ok got it, will change that

Reply:
Done with the change